docs/references/subcommands/release-group/create-release.md

```diff
@@ -0,0 +1,49 @@
+## `fossa release-group create-release`
+
+`fossa release-group create-release` allows you to create a FOSSA release group release.
+
+## Options
+
+Argument              | Required | Description
+----------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------
+`--config` / `-c`     | No       | The to your path to your `.fossa.yml`.
+`--title` / `-t `     | Yes      | The name of your FOSSA release group.
+`--release` / `-r `   | Yes      | The name of your release within your FOSSA release group.
+`--project-locator`   | Yes      | The FOSSA project locator defines a unique ID that the FOSSA API will use to reference this project. The project locator can be found in the UI on the project `Settings` page listed as the `Project Locator` underneath the `Project Title` setting. Specify multiple options by providing this argument multiple times.
+`--project-revision`  | Yes      | The revision associated with your FOSSA project. Project revisions can be found in the UI on the project `Activity` page. Refer to `Revision ID` to retrieve the specific revision you want to use for the project. Specify multiple options by providing this argument multiple times.
+`--project-branch`    | Yes      | The name of the FOSSA project branch. Specify multiple options by providing this argument multiple times.
+
+> NOTE: The arguments listed as `Required` need to be provided through CLI options OR through your `.fossa.yml` configuration.
+
+## .fossa.yml Configuration
+
+Refer to [fossa configuration](../../files/fossa-yml.md) to set up your `.fossa.yml`.
+
+> NOTE: CLI options take precedence over the configurations in `.fossa.yml`.
+
+> NOTE: Project ID values will not work if provided to `releaseGroupProjects.projectLocator`. For more details on the differences between project ID and project locator refer to [documentation](../../files/fossa-yml.md#what-is-the-difference-between-project-id-and-project-locator).
+
+## Example
+
+```bash
+fossa release-group create-release --title example-release-group --release example-release --project-locator custom+1/example --project-revision 1234 --project-branch main --project-locator custom+1/example2 --project-revision 5678 --project-branch main 
```

Comment:
I was able to create a new release group with a branch that did not exist. Is this a bug or is it expected?

```
fossa release-group create-release --title "Test release group" --release example-release-title-3 --project-locator custom+24987/2779 --project-revision 1625011087 --project-branch master --project-locator custom+24987/4361-repro --project-revision 1657063285 --project-branch does-not-exist
```

This successfully created the release. You can see it here on the scottpatten.ca org: https://app.fossa.com/projects/group/1396/projects

When I click through on the "4361-repro" project, I see a "Branch does not exist" error

Reply:
> I was able to create a new release group with a branch that did not exist. Is this a bug or is it expected?

This is expected behavior, I brought it up with Core and filed a ticket: https://fossa.atlassian.net/browse/CORE-3155

Reply:
> I think we should explicitly document that you can have multiple groups of `project-locator`, `project-revision` and `project-branch` args, and that the sets of args are grouped together based on their order.

Good point, I can see this being confusing to readers. There documentation on how those 3 flags are grouped together [here](https://github.com/fossas/fossa-cli/blame/554964d38af1f25c892efd106f8861385d5ca791/docs/references/subcommands/release-group.md#L30) and documentation on providing multiple instances of these flags [here](https://github.com/fossas/fossa-cli/pull/1409/files#diff-951d98f17639116d14a3040694ab216fe4b8311deb470c5b404949c04727a762R12-R14). It's all kind of spread apart so I can write a consolidate blurb as well as adding context on the grouping order.

Reply:
whoa.... all those locator-specific args are required? is there no way to match strings and just create a release called `example-release` within an existing `example-release-group`? (I realize I'm probably oversimplifying) cc @vishal-thenge

Reply:
@ryanlink 

`fossa analyze` is a different work flow. During the time of analyze, we are scanning a particular project, which means that we are able to extract the (project locator, project branch, and project revision) during the time of analysis and that's why they don't need to be explicitly set. 

> we should also decouple creating releases and release groups from project locators, revisions, and branches.

I'm not sure that this is possible, release group releases are inherently associated to projects. The CLI leverages Core's release group endpoints and (project locator, project branch, and project revision) are required by Core's endpoints which is why they are required here.

Reply:
@JeffreyHuynh1 makes sense, although conceptually I can't understand why releases are inherently tied to projects instead of standalone containers. I guess that's why empty releases can't exist, which [CORE-3189](https://fossa.atlassian.net/browse/CORE-3189) was proposed to fix.

Reply:
@ryanlink 100% . When Core has implemented a way to support empty releases, we will be able to remove the requirement of having a project added to a release in the CLI.

Reply:
@zlav 

> do you mean infer the revision and project name that the user wants to add to the release group?
no, I meant in the case where they just want to create a new release version within an RG for subsequent use in a `fossa analyze` command... but wasn't thinking about the fact that you can't currently create an empty release in the core app.

however, this does support Nexthink's request for `fossa analyze --release-group-name XXX -release-group-release ZZZ` to create the release `ZZZ` within the RG if it doesn't already exist.

@JeffreyHuynh1 copying here from Slack - can we add this reference to the docs?
> You can run `fossa analyze --json` and get the data you need https://github.com/fossas/fossa-cli/blob/master/docs/references/subcommands/analyze.md#printing-project-metadata (project locator, etc.)

Reply:
and this may be out of scope for this PR, but echoing @vishal-thenge 's request:

> if we  only have a `projectLocator`; but not: `projectRevision` & `projectBranch` to `fossa release-group`; can we add support to default them to the i) default branch & ii) latest revision ?